pkgs/by-name/ra/rastertoezpl/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  cups,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rastertoezpl";
+  version = "1.1.12";
+
+  src = fetchurl {
+    url = "https://godex.s3-accelerate.amazonaws.com/tWHDYruxWFM98frjWNTE,g.file?v01";
+    name = "${pname}-${version}-source.tar.gz";
+    hash = "sha256-iBhM8Mht/XncWU75cd485WK5GZtJNv78yMsFsD0eKWQ=";
+  };
+
+  buildInputs = [ cups ];
+
+  configureFlags = [
+    "--datarootdir=${placeholder "out"}/share"
+    "PPDDIR=${placeholder "out"}/share/cups/model"
+  ];
+
+  postInstall = ''
+    mkdir -p $out/lib/cups/filter
+    ln -s $out/libexec/rastertoezpl/rastertoezpl $out/lib/cups/filter/rastertoezpl
+  '';
+
+  meta = {
+    description = "CUPS driver for GODEX printers";
+    homepage = "https://www.godexintl.com/downloads";
+    license = lib.licenses.gpl2;
+    maintainers = with lib.maintainers; [ stargate01 ];
+  };
```

Comment:
`meta.platforms` should be added, see https://github.com/NixOS/nixpkgs/blob/0aa475546ed21629c4f5bbf90e38c846a99ec9e9/doc/stdenv/meta.chapter.md#platforms-var-meta-platforms .  As the driver download page advertises the driver "for all linux systems", `lib.platforms.linux` would probably be the correct choice.